source/includes/mongodb-compatibility-table-java.rst

```diff
@@ -184,7 +184,7 @@
      -
      -
      -
-     - |checkmark|  [#driver-support]_
+     - |checkmark|  [#2.14-driver-support]_
```

Comment:
Nitpick: there's an extra space before "The". This doesn't cause errors or change the output, but just want to keep it in the expected format case something changes in the future.